Header finish

iborovikov opened the pull request.

iborovikov committed:
Хедер-без-иконки

iborovikov committed:
Добавил иконку

novikovanat reviewed (commented):
* cursor: pointer; - значение по умолчанию
 * анимацию лучше через миксин делать

Какие измения в свг?

irbis14 commented:
В CSS:
1) строка 45, 106: добавь цвет в утилиты с цветами, чтобы использовать переменную. Цвета очень близкие по оттенку к тем, которые есть в переменных. Проверь, что используешь правильный цвет.
2) добавь анимацию для элементов с ховером и фокусом.

iborovikov committed:
Правки после code review

Добавил в main в 16 строку @import 'utils/transition';.  В утилитный класс .button добавил @include transition(color background-color);.  В политру добавил 2 цвета. Добавил на все интерактивные элементы @include transition()

iborovikov commented:
Добавил в main в 16 строку @import 'utils/transition';. 
В утилитный класс .button добавил @include transition(color background-color);.  
В палитру добавил 2 цвета. 
Добавил на все интерактивные элементы @include transition()
Так же в спрайт добавил 2 иконки (бургер и крестик) так как те что с макета кривые.

iborovikov committed:
button

iborovikov committed:
Merge branch 'master' into header-finish
